### DISCUSSION TITLE
Configure multiple database architecture

### DISCUSSION THREAD

**Discussion by chintankotadia** [statamic/cms#8541] [Troubleshooting] [answered]
Hello Team,

We are building a lighting showcase website using Statamic Pro. Is there a best practice to configure flat files and MySQL simultaneously? We have a limited set of products but a large set of variants that are generated based on different options (Taxonomies). Some insights would be helpful.

PS: We have a well-structured schema in MySQL which works really well with Laravel 8.x but want to migrate it to Statamic now.

Thank you.

**Comment by duncanmcclean** [comment, marked as answer]
If you have an existing database setup, maybe have a look at [Runway](https://github.com/duncanmcclean/runway). 

It'll let you manage your Eloquent models inside the Control Panel, it can handle routing & let you use models using the `{{ runway }}` Antlers tag, etc.

*Disclaimer: I'm the developer behind Runway*

**Comment by chintankotadia** [comment]
Thank you @duncanmcclean! This is really helpful.
